### DISCUSSION TITLE
Korelasi dalam Klasifikasi

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#719] [Q&A] [answered]
Permisi saya ingin bertanya terkait korelasi, jadi pada saat pengerjaan tugas saya melihat statement "Dengan menghapus salah satu kolom yang memiliki korelasi tinggi, kita dapat mengurangi redundansi informasi dalam dataset dan menghindari masalah multikolinearitas" nah untuk hal itu bagaimana ya? apakah memang setiap ada korelasi tinggi dalam klassifikasi kita perlu menghapusnya atau ada kalanya kita biarkan terima kasih

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @daryraa,

Untuk mengetahui apakah kolom-kolom yang digunakan memiliki indikasi multicolinearity (memiliki korelasi tinggi), dapat menggunakan uji VIF dengan cara menginput model ke fungsi `vif(model)`. Jika hasil uji VIF menunjukkan ada kolom-kolom yang memiliki nilai > 10, maka kolom tersebut memiliki korelasi tinggi. Sebaiknya, kolom yang memiliki korelasi tinggi tidak digunakan dalam model Logistic Regression karena akan menurunkan performa model, sehingga salah satu kolomnya perlu dihilangkan.

Semoga menjawab

**Reply by daryraa** [reply]
Apakah hanya untuk logistic regression atau berlaku dengan model lain?

**Reply by arkanayudhistira** [reply]
Meskipun keberadaan multicollinearity memiliki kemungkinan untuk mempengaruhi model-model lain, tapi dampaknya tidak signifikan dibandingkan dengan dampak multicollinearity terhadap model logistic regression dan model-model berbasis regresi lainnya.

**Reply by daryraa** [reply]
Baik mas terima kasih
